Library/Dependencies/napi/CMakeLists.txt

```diff
@@ -2,6 +2,10 @@ cmake_minimum_required(VERSION 3.12)
 
 project(napi)
 
+if (ANDROID)
```

Comment:
I tried to put there 3 lines in root cmakelist.txt but it still tries to compile with Chakra? What did I miss?